src/levelgen_toast.cpp

```diff
@@ -241,6 +241,9 @@ static void expand_process(Level* lvl, PositionQueue& q) {
 	/* Split into one queue per worker. This is relatively cheap. */
 	int worker = 0;
 	auto measure_queue_expand = MeasureFunction<4>{ "expand_process: prepare per-worker queues" };
+	
```

Comment:
Can't really be zero just here as it's global parallelism degree based on processor cores and everything else would break as well. But can be useful to do it as a check on program startup to verify someone didn't modify that constant.

Reply:
Moved assert to main.